docs/conf/scalar-prometheus-custom-values.yaml

```diff
@@ -26,6 +26,13 @@ grafana:
     # requests:
     #   memory: 400Mi
 
+  # -- Grafana's primary configuration
+  grafana.ini:
```

Comment:
This part can be found in
https://github.com/grafana/helm-charts/blob/main/charts/grafana/values.yaml#L577